Cargo.toml

```diff
@@ -31,7 +31,7 @@ debug = true
 ark-ff = { git = "https://github.com/arkworks-rs/algebra/" }
 ark-ec = { git = "https://github.com/arkworks-rs/algebra/" }
 ark-serialize = { git = "https://github.com/arkworks-rs/algebra/" }
-ark-crypto-primitives = { git = "https://github.com/arkworks-rs/crypto-primitives" }
+ark-crypto-primitives = { git = "https://github.com/HungryCatsStudio/crypto-primitives", branch = "absorb-derive"}
```

Comment:
This is temporary until https://github.com/arkworks-rs/crypto-primitives/pull/135 is merged

Reply:
And also https://github.com/arkworks-rs/crypto-primitives/pull/138